Testscripts/Linux/Linux-Test-Project-Tests.sh

```diff
@@ -45,24 +48,38 @@ install_package "${common_packages[@]}"
 case $DISTRO in
     "suse"*)
         suse_packages=(git-core db48-utils libaio-devel libattr1 \
-            libcap-progs libdb-4_8 perl-BerkeleyDB)
+            libcap-progs libdb-4_8 perl-BerkeleyDB ntp)
         install_package "${suse_packages[@]}"
         ;;
     "ubuntu"* | "debian"*)
         deb_packages=(git libaio-dev libattr1 libcap-dev keyutils \
             libdb4.8 libberkeleydb-perl expect dh-autoreconf gdb \
-            libnuma-dev quota genisoimage db-util unzip exfat-utils)
+            libnuma-dev quota genisoimage db-util unzip exfat-utils ntp)
         install_package "${deb_packages[@]}"
         ;;
     "redhat"* | "centos"* | "fedora"*)
-        rpm_packages=(git db48-utils libaio-devel libattr libcap-devel libdb)
+        install_epel
+        rpm_packages=(git db4-utils libaio-devel libattr libcap-devel libdb ntp)
         install_package "${rpm_packages[@]}"
         ;;
     *)
         LogMsg "Unknown distro $DISTRO, continuing to try for RPM installation"
         ;;
 esac
 
+rt_runtime_us=$(cat /sys/fs/cgroup/cpu/user.slice/cpu.rt_runtime_us)
```

Comment:
please add a comment for which LTP test this is required so we don't loose track